Automattic/woocommerce-payments
# Fix placeholder injection problem in New Receipt email

Fixes #4071


#### Changes proposed in this Pull Request
Fixes a bug that causes that the the placeholders `{site_title}` and `{site_url}` in the 'New Receipt' email are not being injected.

The WC_Email parent class merges the placeholders defined in the subclass with its own, when the constructor is executed. For this reason we need to move the call to the parent constructor only after placeholders are defined.










#### Testing instructions





* Set up the local dev store with WCPay plugin to allow for Cash on Delivery orders
* Set up local dev WCPay server to listen to webhooks
* Pay for the order using a card reader, you can emulate one if you follow this [guide](pdjVC1-9M-p2): 
* Check that the `{site_title}` and `{site_url}` placeholders are replaced with the correct data in the email. You can use a plugin like [WP Mail Log](https://wordpress.org/plugins/wp-mail-logging/) for this purpose.

HTML version:

<img width="721" alt="Screen Shot 2022-04-05 at 13 43 48" src="https://user-images.githubusercontent.com/1553182/161746768-a9e3e95d-86cd-4398-8044-3b2fcbbae2a3.png">

-------------------


**Post merge**
